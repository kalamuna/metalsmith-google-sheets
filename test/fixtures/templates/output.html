{{gSheets["Private A Sheet 1"].cells.R1C4.value}}
{{gSheets["Private A Sheet 1"].rows[0].b1}}
