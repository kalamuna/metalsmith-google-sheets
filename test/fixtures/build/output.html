d1
Cell B3
