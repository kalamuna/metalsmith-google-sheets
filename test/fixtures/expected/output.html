Test Phone
Cell B2
